storage/spec/iam_spec.rb

```diff
@@ -0,0 +1,87 @@
+# Copyright 2017 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "../iam"
+require "rspec"
+require "google/cloud/storage"
+require "tempfile"
+
+describe "Google Cloud Storage IAM sample" do
+
+  before do
+    @bucket_name = ENV["GOOGLE_CLOUD_STORAGE_BUCKET"]
+    @storage     = Google::Cloud::Storage.new
+    @project_id  = @storage.project
+    @bucket      = @storage.bucket @bucket_name
+    @test_role   = "roles/storage.admin"
+    @test_member = "user:test@test.com"
+  end
+
+  it "can view bucket IAM members" do
+    @bucket.policy do |policy|
+      policy.roles[@test_role] = [@test_member]
+    end
+
+    @bucket.policy force: true
```

Comment:
Non-blocking for this PR, but I'm curious of your thoughts.  Recommend suggesting different API for google-cloud-storage